settings.gradle

```diff
@@ -126,20 +126,20 @@ include ':instrumentation:play-ws:play-ws-1.0'
 include ':instrumentation:play-ws:play-ws-2.0'
 include ':instrumentation:play-ws:play-ws-2.1'
 include ':instrumentation:play-ws:play-ws-common'
-include ':instrumentation:rabbitmq-amqp-2.7'
+include ':instrumentation:rabbitmq-2.7'
 include ':instrumentation:ratpack-1.5'
-include ':instrumentation:reactor-3.1'
+include ':instrumentation:project-reactor-3.1'
```

Comment:
sorry, i was later convinced that `reactor` alone was sufficient, but didn't update that issue